javascript/code-challenges/challenges-02.test.js

```diff
@@ -0,0 +1,309 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+
+CHALLENGE 1 - Review
+
+Write a function named raisedToTheThird that takes in an array of numbers and returns a new array of each of those numbers raised to the 3rd power (hint: look up Math.pow()). Use forEach to solve this problem.
+
+------------------------------------------------------------------------------------------------ */
+
+const raisedToTheThird = (arr) => {
+  const newArr = arr.map(num => Math.pow(num, 3));
+  // return newArr;
+  return newArr;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named addOne that, given an array of numbers, uses map to return a new array with each value simply incremented by 1. 
+------------------------------------------------------------------------------------------------ */
+
+const addOne = (arr) => {
+  const newArr = arr.map(num => num + 1);
+  return newArr;
+};
```

Comment:
Map automatically create a new array. So you could just return map
```js
return arr.map(num => num + 1);
```
And while you're at it you could even do an implicit return to make it a one line answer
```js
const addOne = arr => arr.map(num => num + 1);
```